src/ontology/mondo-edit.obo

```diff
@@ -31153,11 +31153,15 @@ property_value: IAO:0000589 "bacteriuria (disease)" xsd:string
 id: MONDO:0001883
 name: blue toe syndrome
 def: "A condition that is caused by recurring atheroembolism in the lower extremities. It is characterized by cyanotic discoloration of the toes, usually the first, fourth, and fifth toes. Discoloration may extend to the lateral aspect of the foot. Despite the gangrene-like appearance, blue toes may respond to conservative therapy without amputation." [MESH:D018438]
+synonym: "purple toe syndrome" EXACT [DOID:1461]
+synonym: "trash foot" EXACT [DOID:1461]
+synonym: "warfarin blue toe syndrome" EXACT [DOID:1416]
```

Comment:
@yshwetar these synonyms should not be on this term because DOID:1461 is not an equivalent mapping